script/tool/lib/src/common/pub_version_finder.dart

```diff
@@ -27,10 +27,10 @@ class PubVersionFinder {
 
   /// Get the package version on pub.
   Future<PubVersionFinderResponse> getPackageVersion(
-      {required String package}) async {
-    assert(package.isNotEmpty);
+      {required String packageName}) async {
+    assert(packageName.isNotEmpty);
```

Comment:
fyi: It would be nice if Dart could statically assert this.  In many places in my code I just traded null checks for isEmpty checks.

Reply:
> I just traded null checks for isEmpty checks

Well, not really; you traded code that was wrong without you noticing for code that is correct 🙂. Because if an empty string wasn't valid, but the check you had before was `foo != null` rather than `foo != null && !foo.isEmpty`, then the old code was wrong.